### ISSUE TITLE
How manually use lucky egg

### ISSUE THREAD

**Issue by pokehoang75** [Lunat1q/Catchem-PoGo#1435]
Just a question or if we don't have it yet, would it be a future feature request.

Thank You all for your hard working!

**Comment by petoe93** [comment]
this was discussed like 1000 times already..just use the search function before starting a new issue..or check the official discord channel

**Comment by pokehoang75** [comment]
Thanks I got it now, I'm nb here with github.
Closed!
